scripts/sysdig-probe-loader

```diff
@@ -21,6 +21,9 @@ get_kernel_config() {
 	elif [ ! -z "${SYSDIG_HOST_ROOT}" ] && [ -f "${SYSDIG_HOST_ROOT}/usr/lib/ostree-boot/config-${KERNEL_RELEASE}" ]; then
 		echo "Found kernel config at ${SYSDIG_HOST_ROOT}/usr/lib/ostree-boot/config-${KERNEL_RELEASE}"
 		HASH=$(md5sum "${SYSDIG_HOST_ROOT}/usr/lib/ostree-boot/config-${KERNEL_RELEASE}" | cut -d' ' -f1)
+	elif [ -f /usr/lib/modules/${KERNEL_RELEASE}/config ]; then
```

Comment:
In order for this to work in a container, we should need another elif that checks for `"${SYSDIG_HOST_ROOT}/usr/lib/modules/${KERNEL_RELEASE}/config"` like we do for `/boot/config` and `/usr/lib/ostree-boot/config`. How is this working without that?

Reply:
inside the container, /lib/modules is symlink of $SYSDIG_HOST_ROOT/lib/modules. w/ such symlink in place, the code applies to both native host and container.
see sysdig/docker/local/Dockerfile line 46
 RUN ln -s $SYSDIG_HOST_ROOT/lib/modules /lib/modules

Reply:
From slack, we found that this works because `/lib` is symlinked to `/usr/lib` in that base image and we set up symlinks in `/lib/modules` to `/$SYSDIG_HOST_ROOT/lib/modules` in the docker entrypoint.

To make this robust, let's check `/lib/modules/${KERNEL_RELEASE}/config` and the same path prefixed with `/$SYSDIG_HOST_ROOT`. That will still work for the atomic container and will be more flexible in other cases.